siliconcompiler/core.py

```diff
@@ -162,6 +162,20 @@ def top(self):
             return self.design
         return entrypoint
 
+    ###########################################################################
+    def get_task(self, step, index='0'):
```

Comment:
might be nice to add flow as an input as well, maybe `flow=None` and use `option,flow` if `None`